Wikipedia/Code/WMFFeedImage.m

```diff
@@ -26,8 +29,8 @@ + (NSValueTransformer *)imageThumbURLJSONTransformer {
                                               BOOL *success,
                                               NSError *__autoreleasing *error) {
             NSInteger sizePrefix = WMFParseSizePrefixFromSourceURL(urlString);
-            if (sizePrefix < 640) {
-                urlString = WMFChangeImageSourceURLSizePrefix(urlString, 640);
+            if (sizePrefix < WMFImageWidthExtraLarge) {
+                urlString = WMFChangeImageSourceURLSizePrefix(urlString, WMFImageWidthExtraLarge);
```

Comment:
Is the intent here just to have a more sensible default given the prevalence of 2x scale and above devices?

Reply:
Yup! And to use a defined constant so if those thumbnail widths ever change it'd be updated here